setup.py

```diff
@@ -91,6 +91,10 @@ def pre_post_plugin_type():
         requirements.append("urllib3<=1.24.3")
         requirements.append("stevedore>=1.8.0,<=1.10.0")
         requirements.append("avocado-framework>=68.0,<70.0")
+        requirements.append("python-passfd")
+        # we need these three packages to have python-passfd installed successfully
+        if os.system('rpm -q gcc python2-devel redhat-rpm-config'):
+            os.system('yum install gcc python2-devel redhat-rpm-config -y')
```

Comment:
`setup.py` is a generic file, not tied to any distribution. Let's just add the `python-passfd` requirement and leave it to users to configure their systems accordingly (feel free to suggest the dependencies a comment, that's the best you can do)

Reply:
Got it,thanks for your guide.